include/zephyr/bluetooth/mesh/msg.h

```diff
@@ -31,12 +31,22 @@ struct bt_mesh_model;
 /** Length of a long Mesh MIC. */
 #define BT_MESH_MIC_LONG 8
 
+/** 1-octet opcode. */
+#define BT_MESH_MODEL_OP_LEN_1	1
+/** 2-octets opcode. */
+#define BT_MESH_MODEL_OP_LEN_2	2
+/** 3-octets opcode. */
+#define BT_MESH_MODEL_OP_LEN_3	3
```

Comment:
Are these macro really needed?

Reply:
Adding isn't a bad thing, at least it's more readable.

Reply:
Instead of reading number I need to read the macro name, then go to the declaration to see what it expands to.

Reply:
I'd recommend to remove these macros.

Reply:
Okay,